deploy/charts/openebs-monitoring/rules/volumes/volume-node-rules.yaml

```diff
@@ -0,0 +1,47 @@
+groups:
+  - name: volume-node
+    rules:
+      # Alert for pv's fileystem getting into read-only state while re-mounting on a particular node.
+      - alert: VolumeNodeFileSystemIsReadOnly
+        annotations:
+          summary: 'Persistent Volume on node ''{{ $labels.node }}'' for persistent volume claim ''{{ $labels.persistentvolumeclaim }}'' is having read-only filesystem while re-mounting'
+          description: |-
+            Persistent Volume's filesystem on node '{{ $labels.node }}' for persistent volume claim '{{ $labels.persistentvolumeclaim }}' has become read-only
+        expr: |-
+          kubelet_volume_stats_inodes * on(node) group_left(reason) problem_counter{reason="FilesystemIsReadOnly"} > 0
+        for: 5m
+        labels:
+          severity: "critical"
+      # Alert for pv's facing issues with ext4 filesystem on a particular nodes.
+      - alert: VolumeNodeExt4Error
+        annotations:
+          summary: 'Persistent Volume on node ''{{ $labels.node }}'' for persistent volume claim ''{{ $labels.persistentvolumeclaim }}'' is having issues with ext4 filesystem'
+          description: |-
+            Persistent Volume's on node '{{ $labels.node }}' persistent volume claim '{{ $labels.persistentvolumeclaim }}' encountering ext4 filesystem error
+        expr: |-
+          kubelet_volume_stats_inodes * on(node) group_left(reason) problem_counter{reason="Ext4Error"} > 0
+        for: 5m
+        labels:
+          severity: "critical"
+      # Alert for pv's facing issues with I/O on a particular node.
+      - alert: VolumeNodeIOError
+        annotations:
+          summary: 'Persistent Volume on node ''{{ $labels.node }}'' for persistent volume claim ''{{ $labels.persistentvolumeclaim }}'' is having trouble with I/O'
+          description: |-
+            Persistent Volume on node '{{ $labels.node }}' for persistent volume claim '{{ $labels.persistentvolumeclaim }}' encountering errors w.r.t buffer I/O 
```

Comment:
what is buffer IO? How was that detected?

Reply:
All these comes from system logs. The same failure event as above generated the error.

Reply:
Do we need different alerts for the same issue?

Reply:
Yup. Every condition for which that volume is creating a problem.

Reply:
Hmm..ok.